### DISCUSSION TITLE
Possibility of hosting the docker file on dockerhub?

### DISCUSSION THREAD

**Discussion by BasadoBased** [danny-avila/LibreChat#1761] [Feature Requests & Suggestions]
Just curious if its being considered. You should be able to add it to the ci pipeline and it makes installation with docker a bit easier.

**Comment by danny-avila** [comment]
Thanks I will look into it. I used to early on in the project but you're right it could be simple to implement

**Reply by danny-avila** [reply]
@BasadoBased https://github.com/danny-avila/LibreChat/pull/1762 will merge later today

**Reply by BasadoBased** [reply]
Incredible, thank you.

**Reply by danny-avila** [reply]
Looks like it ran successfully (dev images)! https://github.com/danny-avila/LibreChat/actions/runs/7855979908/job/21438269800

https://hub.docker.com/repositories/librechat
